From: Russell Coker <russell@coker.com.au>
Date: Mon, 21 Jun 2010 22:35:00 +1000
Subject: Made fixfiles display the progress

---
 scripts/fixfiles |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/fixfiles b/scripts/fixfiles
index 27dcccf..310fccf 100755
--- a/scripts/fixfiles
+++ b/scripts/fixfiles
@@ -207,12 +207,12 @@ if [ ! -z "$PREFC" ]; then
 fi
 if [ ! -z "$RPMFILES" ]; then
     for i in `echo "$RPMFILES" | sed 's/,/ /g'`; do
-	rpmlist $i | ${RESTORECON} ${FORCEFLAG} $* -R -i -f - 2>&1 | cat >> $LOGFILE
+	rpmlist $i | ${RESTORECON} ${FORCEFLAG} $* -R -i -p -f - 2>&1 | cat >> $LOGFILE
     done
     exit $?
 fi
 if [ ! -z "$FILEPATH" ]; then
-    ${RESTORECON} ${FORCEFLAG} -R $* $FILEPATH 2>&1 | cat >> $LOGFILE
+    ${RESTORECON} ${FORCEFLAG} -R -p $* $FILEPATH 2>&1 | cat >> $LOGFILE
     return
 fi
 [ -x /usr/sbin/genhomedircon ] && /usr/sbin/genhomedircon
