From: Laurent Bigonville <bigon@debian.org>
Date: Mon, 5 Mar 2012 22:45:58 +0100
Subject: fix ftbfs with hardening flags

---
 sandbox/seunshare.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sandbox/seunshare.c b/sandbox/seunshare.c
index c493e98..d7eb0b9 100644
--- a/sandbox/seunshare.c
+++ b/sandbox/seunshare.c
@@ -311,8 +311,8 @@ static int seunshare_mount(const char *src, const char *dst, struct stat *src_st
  */
 static int sandbox_error(const char *string)
 {
-	fprintf(stderr, string);
-	syslog(LOG_AUTHPRIV | LOG_ALERT, string);
+	fprintf(stderr, "%s", string);
+	syslog(LOG_AUTHPRIV | LOG_ALERT, "%s", string);
 	exit(-1);
 }
 
