From: Russell Coker <russell@coker.com.au>
Date: Thu, 15 Mar 2012 10:52:29 +1100
Subject: Remove non-existing sucess and failure functions
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=663419

---
 sandbox/sandbox.init |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/sandbox/sandbox.init b/sandbox/sandbox.init
index 78459a1..b235e3d 100644
--- a/sandbox/sandbox.init
+++ b/sandbox/sandbox.init
@@ -62,20 +62,20 @@ status() {
 
 case "$1" in
     restart)
-	start && success || failure
+	start
 	;;
 
     force-reload)
-	start && success || failure
+	start
 	;;
 
     start)
-	start && success || failure
+	start
 	echo
 	;;
 
     stop)
-	stop && success || failure
+	stop
 	echo
 	;;
 
